src/server/services/Actions.ts

```diff
@@ -45,24 +38,21 @@ export default class Actions {
       waitReady().then(() => {
         this.account = keyring.addFromMnemonic(mnemonic);
 
-        this.updateFaucetBalance().then(() =>
-          logger.info('Fetched faucet balance 💰')
-        );
         // We do want the following to just start and run
         // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
Does this matter anymore?

Reply:
removed [c513bbb](https://github.com/paritytech/substrate-matrix-faucet/pull/147/commits/c513bbba76ffeddfc0a5022380e7179fc87e2d97)